cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/AuthorizationBootstrapperTest.java

```diff
@@ -108,6 +108,8 @@ public static void setup() throws Exception {
     cConf.setBoolean(Constants.Security.ENABLED, true);
     cConf.setBoolean(Constants.Security.KERBEROS_ENABLED, false);
     cConf.setBoolean(Constants.Security.Authorization.ENABLED, true);
+    cConf.set(Constants.Security.CFG_CDAP_MASTER_KRB_PRINCIPAL,
+              UserGroupInformation.getCurrentUser().getShortUserName());
```

Comment:
Let's change this to `getLoginUser()`. Note, this will need a corresponding change to `Principal systemUser` in method `test()`